### ISSUE TITLE
Multiple result base

### ISSUE THREAD

**Issue by TheNicker** [Qalculate/qalculate-gtk#490]
Currently the result base is only one of many to choose from: hex,dec,oct,bin, etc..
It would be very convenient to view a number in multiple bases simultaneously like windows calc when in programmer mode
<img width="284" alt="image" src="https://github.com/Qalculate/qalculate-gtk/assets/4753283/e6b5c593-b0ef-4181-9e8d-a924ee0eb1d4">

**Comment by hanna-kn** [comment]
<img width="452" alt="Screenshot_20230822_162202" src="https://github.com/Qalculate/qalculate-gtk/assets/19703699/f9903adb-5441-4401-96da-f186e520a8cc">
<img width="506" alt="Screenshot_20230822_162558" src="https://github.com/Qalculate/qalculate-gtk/assets/19703699/2709fcef-fc33-4aaf-8c6f-0e3fbf877596">
<img width="321" alt="Screenshot_20230822_162708" src="https://github.com/Qalculate/qalculate-gtk/assets/19703699/dc072be2-18fd-4a68-94f1-1a21578d647f">

**Comment by TheNicker** [comment]
Hi @hanna-kn ,
First, thank you for this software, it helped me in my recent project.

I can see that you showcased the Qt version, and I also noticed that except libqalculate, the Qt and GTK versions are completely different.

I was using the GTK version, so by pressing CTRL+B, I got the number conversion window.  However it is sepereated from the main window and doesn't update on-the-fly as in the Qt version

I have no problem running both the Qt and the GTK version under gnome, though ideally only one flavour should be used. 
It seems like the GTK version is more popular and richer in features, so it'll be helpful to get 
the full featured multi result-base into it.

**Comment by hanna-kn** [comment]
I mainly intended to show the multiple number bases displayed above the programming keypad (Ctrl+P), in the first screenshot. The number bases view in the Qt version is nicer, but I've not been able to design anything similar which fits the main window of the GTK version.

**Comment by TheNicker** [comment]
I'm trying to understand the issue. 
Is it a GTK technical issue or visual design decision that needs to take place, or else?

**Comment by hanna-kn** [comment]
It is primarily a visual design issue.

**Comment by TheNicker** [comment]
In these scenarios, personally I prefer functionality over aesthetics.

It's OK if it's not perfect, worst case it can be opted out by default

**Comment by hanna-kn** [comment]
> In these scenarios, personally I prefer functionality over aesthetics.

The programming keypad in qalculate-gtk provides the same functionality, if calculate-as-you-type is activated, but the visual design of the bases dock window in qalculate-qt improves the usability.

**Comment by TheNicker** [comment]
> The programming keypad in qalculate-gtk provides the same functionality

Last time I checked, "Number Bases" window doesn't update while typing even if "caluclate as you type" is activated.
It needs to be reopened or refreshed by CTRL+B

**Comment by hanna-kn** [comment]
> > The programming keypad in qalculate-gtk provides the same functionality
> 
> Last time I checked, "Number Bases" window doesn't update while typing even if "caluclate as you type" is activated. It needs to be reopened or refreshed by CTRL+B

That is true, the number bases window does not update automatically when the main expression entry or result is updated (the number bases window were mainly intended for separate use).

I'm however referring to the alternative "programming keypad in the main window, opened with Ctrl+P or the left-most button at the top of the keypad.
